### PR TITLE
Insert With Select

### PR DESCRIPTION
**Pull request by fulviocanducci**
### Example:

```
using (var connection = _databaseFixture.Connection)
{
	connection.Open();
	var rows = connection.Insert()
		.Into("Test2")
		.Columns("Name")                      
                .Values(x => connection.Select("Name").From("Test1"))  // IQueryCommand
		.Execute();
}
```

### CODE REVIEW

File: Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs [hunk 1]
```diff
@@ -15,17 +15,20 @@ public DatabaseFixture()
                 .CreateProfileTable()
                 .CreateUserTable()
                 .CreatePeopleTable()
+                .CreateTest1AndTest2Table()
```
Comment (gustavoferrazfontes) [inline Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs]: Please, create a better method name. Some more meaning.

File: Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs [hunk 2]
```diff
@@ -15,17 +15,20 @@ public DatabaseFixture()
...
                 .LoadProfileTable()
                 .LoadUserTable()
-                .LoadPeopleTable();
+                .LoadPeopleTable()
+                .LoadTest1Table();
```
Comment (gustavoferrazfontes) [inline Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs:22]: Please, create a better method name. Some more meaning.

File: Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs [hunk 3]
```diff
@@ -15,17 +15,20 @@ public DatabaseFixture()
...
         }
 
         public void Dispose()
         {
             Connection
                 .ResetUserTable()
                 .ResetProfileTable()
-                .ResetPeopleTable();
+                .ResetPeopleTable()
+                .ResetTest1AndTest2Table();
```
Comment (gustavoferrazfontes) [inline Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs]: Please, create a better method name. Some more meaning.

File: Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs (hunk 1 above)
Comment (fulviocanducci) [reply Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs]: Any suggestion?

File: Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs (hunk 2 above)
Comment (fulviocanducci) [reply Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs:22]: Any suggestion?

File: Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs (hunk 3 above)
Comment (fulviocanducci) [reply Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs]: Any suggestion?

File: Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs (hunk 1 above)
Comment (gustavoferrazfontes) [reply Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs]: Maybe `CreateTablesForInsertIntoTests`

Comment (fulviocanducci) [reply Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs]: Ok

File: Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs (hunk 2 above)
Comment (fulviocanducci) [reply Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs:22]: Rename

File: Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs (hunk 3 above)
Comment (fulviocanducci) [reply Flepper.Tests.Integration/Infra/Fixture/DatabaseFixture.cs]: Rename

**Review by gustavoferrazfontes — APPROVED** [review]

**Pull request merged**